recipes-sota/aktualizr/aktualizr-auto-prov.bb

```diff
@@ -40,7 +39,7 @@ do_install_append() {
         install -d ${D}/${systemd_unitdir}/system
         install -m 0644 ${WORKDIR}/aktualizr-autoprovision.service ${D}/${systemd_unitdir}/system/aktualizr.service
         install -d ${D}${libdir}/sota
-        install -m "0644" ${WORKDIR}/sota_autoprov.toml ${D}${libdir}/sota/sota.toml
+        install -m "0644" ${STAGING_DIR_NATIVE}/${libdir}/sota/sota_autoprov.toml ${D}${libdir}/sota/sota.toml
```

Comment:
What about creating a symlink to the thing installed by the aktualizr recipe? Adding a dependency on aktualizr-native feels a bit weird

Reply:
You mean install all of the *.toml files on every device and then symlink to the correct one?